distribution/lib/Standard/Base/0.0.0-dev/src/Meta/Enso_Project.enso

```diff
@@ -47,3 +49,29 @@ type Project_Description
              enso_project.namespace
     namespace : Text
     namespace self = self.prim_config.namespace
+
+## Returns the Enso project description for the given module.
+
+   Arguments:
+   - module: An optional module for which the Enso project description
+     will be fetched. If `Nothing`, the current project description will
+     be fetched.
+
+   > Example
+     Get the project description for the current project.
+
+         enso_project
```

Comment:
Can we clarify what does `current project` mean?

If I have:
```
Project_1/
    src/
        Main.enso:
             from Standard.Base import all
             foo = IO.println enso_project
Project_2/
    src/
        Main.enso:
             import local.Project_1
             main = Project_1.foo
```

will I get `Project_1` or `Project_2`?

Reply:
In your example it would return `Project_2`. Docs updated in 6a5f147ce8c1428712b20fde55a9ae6cd40e54e8

Reply:
Awesome.